packages/back-end/src/services/experimentNotifications.ts

```diff
@@ -132,29 +127,20 @@ export const notifyAutoUpdate = ({
 export const notifyMultipleExposures = async ({
   context,
   experiment,
-  results,
-  snapshot,
+  currentStatus,
 }: {
   context: Context;
   experiment: ExperimentInterface;
-  results: ExperimentReportResultDimension;
-  snapshot: ExperimentSnapshotDocument;
+  currentStatus: RunningExperimentStatusData;
 }) => {
-  const totalUsers = results.variations.reduce(
-    (totalUsersCount, { users }) => totalUsersCount + users,
-    0
-  );
-
-  const multipleExposureHealth = getMultipleExposureHealthData({
-    multipleExposuresCount: snapshot.multipleExposures,
-    totalUsersCount: totalUsers,
-    minCountThreshold: DEFAULT_MULTIPLE_EXPOSURES_ENOUGH_DATA_THRESHOLD,
-    minPercentThreshold:
-      context.org.settings?.multipleExposureMinPercent ??
-      DEFAULT_MULTIPLE_EXPOSURES_THRESHOLD,
-  });
-
-  const triggered = multipleExposureHealth.status === "unhealthy";
+  if (currentStatus.status !== "unhealthy") {
+    return;
+  }
+  const multipleExposureData = currentStatus.unhealthyData.multipleExposures;
+  if (!multipleExposureData) {
+    return;
+  }
+  const triggered = true;
```

Comment:
suggestion: Should we update this to check if this is a new status or if the experiment was already unhealthy?

I think that was the original intent of this variable (as in previously triggered).

Reply:
I think that's what the memoization does, but now that you mention it, I think I realize that in order for the `memoizeNotification` to remove this unhealthy status if it flicks off, I need to not `return` early.